app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -218,17 +222,36 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         themeManager.applyStatusBarTheme(this)
     }
 
-    private fun setupToolbarAndNavigation() {
+    fun getSupportActionBarAndInflateIfNecessary(): Toolbar? {
```

Comment:
nit: we should add some kdoc because why you might want to use this method is not clear. The first sentence in kdoc is always what the thing does (even if it's obvious) because it's used as a summary in the generated kdoc. The remaining sentences, in this case, can be used to say something like, "Callers trying to get the [supportActionBar] from the Activity should call this instead." and maybe a brief explanation why (b/c ViewStub).

Note kdoc uses 2 starts to start:
```kotlin
/**
 * kdoc block
 */
```